example_client/benchmark/client.py

```diff
@@ -0,0 +1,470 @@
+#
+# INTEL CONFIDENTIAL
+# Copyright (c) 2021 Intel Corporation
+#
+# The source code contained or described herein and all documents related to
+# the source code ("Material") are owned by Intel Corporation or its suppliers
+# or licensors. Title to the Material remains with Intel Corporation or its
+# suppliers and licensors. The Material contains trade secrets and proprietary
+# and confidential information of Intel or its suppliers and licensors. The
+# Material is protected by worldwide copyright and trade secret laws and treaty
+# provisions. No part of the Material may be used, copied, reproduced, modified,
+# published, uploaded, posted, transmitted, distributed, or disclosed in any way
+# without Intel's prior express written permission.
+#
+# No license under any patent, copyright, trade secret or other intellectual
+# property right is granted to or conferred upon you by disclosure or delivery
+# of the Materials, either expressly, by implication, inducement, estoppel or
+# otherwise. Any license under such intellectual property rights must be express
+# and approved by Intel in writing.
+#
+
+"""Base inference client for benchmarks"""
+
+import os
+import io
+import abc
+import sys
+import time
+import math
+import json
+import random
+import traceback
+
+import png
+import grpc
+import numpy
+import tensorflow
+from retry.api import retry_call
+from ovms_benchmark_client.metrics import XSeries
+
+
+class BaseClient(metaclass=abc.ABCMeta):
+
+    STATEFUL_WIP = 0
+    STATEFUL_START = 1
+    STATEFUL_STOP = 2
+
+    indent = 1 * " "
+    indent2 = 2 * " "
+    json_prefix = "JSON-OUTPUT:"
+    grpc_options = [
+        ("grpc.max_message_length", 1024 * 1024 * 1024),
```

Comment:
I wonder if we could drop this line? 
`max_message_length` is deprecated - https://grpc.github.io/grpc/core/deprecated.html#_deprecated000001
I think `max_send_message_length` and `max_receive_message_length` will do the job